packages/babel-preset/namespace-plugin.js

```diff
@@ -104,14 +104,18 @@ module.exports = function namespacePlugin({ types: t }) {
             return;
           }
 
-          const idProp = messageObj
-            .get('properties')
-            .find(p => p.get('key').node.name === 'id')
-            .get('value');
-
-          if (idProp && !idProp.node.value.startsWith(prefix)) {
-            idProp.replaceWith(
-              t.StringLiteral(`${prefix}${idProp.node.value}`), // eslint-disable-line new-cap
+          const idProp = messageObj.get('properties').find(p => {
```

Comment:
probably want to separate this out into a separate bugfix commit re: supporting inlined message definitions